src/server/snapshot.cc

```diff
@@ -159,8 +159,8 @@ void SliceSnapshot::IterateBucketsFb(const Cancellation* cll) {
 
   // TODO: investigate why a single byte gets stuck and does not arrive to replica
   for (unsigned i = 10; i > 1; i--)
-    CHECK(!default_serializer_->SendFullSyncCut());
-  FlushDefaultBuffer(true);
+    CHECK(!serializer_->SendFullSyncCut());
+  PushSerializedToChannel(true);
```

Comment:
😅

Reply:
OMG, maybe we do not? I suggest removing this loop and see if the issue remains.

Reply:
We still have bug here